languages/lt.json

```diff
@@ -1,360 +1,515 @@
+      "cooldown": "Įjungtas 5 sekundžių vėlavimas žinutėms",
+      "open_chat": "Išplėsti tiesioginį pokalbį"
+    },
+    "feed": {
+      "empty_feed_title": "Kol kas neturime ką parodyti...",
+      "empty_feed_details": "Eikite ir tyrinėkite pagrindinį puslapį, tada sekite patinkančius kūrėjus ir čia bus rodomi jų naujausi vaizdo įrašai ir pranešimai!"
+    },
+    "login": {
+      "yes_account": "Jau turite paskyrą?",
+      "enthusiastic_login": "Prisijunkite dabar!",
+      "no_account": "Dar neturite paskyros?",
+      "enthusiastic_register": "Užsiregistruokite dabar!",
+      "forgot_password": "Pamiršote slaptažodį?",
+      "enthusiastic_reset_password": "Nustatykite jį iš naujo čia!"
+    },
+    "posts": {
+      "empty": "Šiame profilyje dar nėra jokių pranešimų.",
+      "blank_title": "Negalite turėti tuščio pavadinimo.",
+      "post_title": "Pranešimo pavadinimas",
+      "post_text": "Pranešimo tekstas",
+      "edit_post": "Redaguoti pranešimą",
+      "edit_success": "Redaguotas pranešimas",
+      "create_post": "Sukurti naują pranešimą",
+      "post": "Pranešimas",
+      "posts": "Pranešimai",
+      "delete_success": "Sėkmingai ištrintas pranešimas",
+      "delete_fail": "Pranešimo nepavyko ištrinti, pabandykite dar kartą",
+      "confirm_deletion": "Ar tikrai norite ištrinti šį pranešimą?",
+      "could_not_delete": "Nepavyko ištrinti pranešimo, nėra aktyvaus profilio",
+      "could_not_edit": "Nepavyko redaguoti pranešimo, nėra aktyvaus profilio",
+      "could_not_create": "Nepavyko sukurti pranešimo, nėra aktyvaus profilio",
+      "create_success": "Sėkmingai sukurtas pranešimas",
+      "comments_none": "Jūs pasiekėte šio pranešimo komentarų pabaigą"
+    },
+    "categories": {
+      "category": "Kategorija",
+      "comedy": "Komedija",
+      "commentary": "Komentarai",
+      "education": "Edukacija",
+      "entertainment": "Pramogos",
+      "film_and_animation": "Kinas ir animacija",
+      "gaming": "Žaidimai",
+      "health_and_beauty": "Sveikata ir grožis",
+      "how_to": "Kaip padaryti",
+      "music": "Muzika",
+      "people_blogs_and_vlogs": "Žmonės, dienoraščiai ir vaizdo tinklaraštis",
+      "pets_and_animals": "Gyvūnai ir augintiniai",
+      "science_and_technology": "Mokslas ir technologijos",
+      "sports": "Sportas"
+    },
+    "comments": {
+      "modal": {
+        "delete_comment_title": "Ar tikrai norite ištrinti šį komentarą?"
+      },
+      "errors": {
+        "blank": "Negalima paskelbti tuščio komentaro"
+      },
+      "none": "Komentarų šiam vaizdo įrašui nėra.",
+      "linked": "Susijęs komentaras",
+      "pinned_by": "Prisegtas",
+      "upvoted_by": "Patinka",
+      "edited": "redaguotas",
+      "show_more_replies": "Rodyti daugiau atsakymų",
+      "show_all_replies": "Rodyti visus atsakymų",
+      "pin": "Prisegti"
+    },
+    "creator_portal": {
+      "creator_news": "Kūrėjų naujienos",
+      "creator_news_soon": "Creator News is coming soon!",

```

Comment:
I assume this was an accident? Your previous commit had translations for creator_news, creator_news_soon, etc